pdfmaker/font_reader.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python
+from reportlab.pdfbase import pdfmetrics
+from reportlab.pdfbase.ttfonts import TTFont
+from fontTools import ttLib
+from glob import glob
+
+
+def registerFonts():
+    rootDir = "fonts"
+
+    for fontLoc in glob(rootDir + "/*.ttf"):
```

Comment:
If they have no ttf files in their font directory, it looks like this won't throw up an error and most likely blow up when it starts trying to draw glyphs.